rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -25,3 +47,95 @@ def __init__(
             pool_address=POOL_ADDRESS,
             eth_gateways=ETH_GATEWAYS,
         )
+
+    def _decode_staking_events(self, context: DecoderContext) -> DecodingOutput:
+        """Decode aave v2 staking unstaking events"""
+        if context.tx_log.topics[0] not in {STAKED_AAVE, REDEEM_AAVE}:
+            return DEFAULT_DECODING_OUTPUT
+
+        from_address = hex_or_bytes_to_address(context.tx_log.topics[1])
+        to_address = hex_or_bytes_to_address(context.tx_log.topics[2])
+        if not self.base.any_tracked([from_address, to_address]):
+            return DEFAULT_DECODING_OUTPUT
+
+        amount = token_normalized_value_decimals(
+            token_amount=hex_or_bytes_to_int(context.tx_log.data[:32]),
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )
+        if context.tx_log.topics[0] == STAKED_AAVE:
+            return self._decode_stake(to_address, amount, context.decoded_events)
+
+        return self._decode_unstake(to_address, amount, context.decoded_events)
+
+    def _decode_stake(
+            self,
+            to_address: ChecksumEvmAddress,
+            amount: FVal,
+            decoded_events: list[EvmEvent],
+    ) -> DecodingOutput:
+        """Decode aave v2 stake event"""
```

Comment:
I won't repeat it anymore but do not mention v2 here. In fact this is such a small function no need for a docstring